cicd/pipelines/securing-webhooks-with-event-listeners.adoc

```diff
@@ -0,0 +1,22 @@
+[id="securing-webhooks-with-event-listeners"]
+= Securing webhooks with event listeners
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+:context: securing-webhooks-with-event-listeners
+
+toc::[]
+
+Triggers in {pipelines-title} support both HTTP (insecure) and HTTPS (secure HTTP) connections to the `Eventlistener` resource. With the secure HTTPS connection, you get an end-to-end secure connection within and outside the cluster. After you create a namespace, you can enable this secure HTTPS connection for the `Eventlistener` resource by adding the `operator.tekton.dev/enable-annotation=enabled` label to the namespace, and then creating a `Trigger` resource and a secured route using re-encrypt TLS termination.
+
+When an OpenShift pipeline is installed through an operator a `tekton-operator-proxy-webhook-` pod will always be running. The webhook watches for labels on the namespace and the below annotation is set on the `Eventlistener` object, whenever one is found, which in turn creates secrets and required certificates:
```

Comment:
When you install a {pipelines-title} using the operator, a `tekton-operator-proxy-webhook` pod always runs in the background. The webhook watches for labels in the namespace, and the `service.beta.openshift.io/serving-cert-secret-name=<secret_name>` annotation is set on the `EventListener` object, whenever one is found. This, in turn, creates secrets and required certificates.

COMMENT - Even with this rephrasing, I am not satisfied. The technical meaning of the sentences are not clear. Try using active voice as much as possible, with a clear subject-verb-object narrative.